cmake/modules/FindGINKGO.cmake

```diff
@@ -0,0 +1,72 @@
+## ---------------------------------------------------------------------
+##
+## Copyright (C) 2018 by the deal.II authors
+##
+## This file is part of the deal.II library.
+##
+## The deal.II library is free software; you can use it, redistribute
+## it, and/or modify it under the terms of the GNU Lesser General
+## Public License as published by the Free Software Foundation; either
+## version 2.1 of the License, or (at your option) any later version.
+## The full text of the license can be found in the file LICENSE.md at
+## the top level directory of deal.II.
+##
+## ---------------------------------------------------------------------
+
+#
+# Try to find the GINKGO library
+#
+# This module exports
+#
+#   GINKGO_INCLUDE_DIRS
+#
+
+SET(GINKGO_DIR "" CACHE PATH "An optional hint to a GINKGO installation")
+SET_IF_EMPTY(GINKGO_DIR "$ENV{GINKGO_DIR}")
+
+DEAL_II_FIND_LIBRARY(GINKGO_LIBRARY
```

Comment:
We typically put these details inside a module, which in this case should be under `cmake/modules/FindGINKO.cmake`. Please see this example ([module file](https://github.com/dealii/dealii/blob/master/cmake/modules/FindMETIS.cmake); [configure file](https://github.com/dealii/dealii/blob/master/cmake/configure/configure_2_metis.cmake)) to see how its normally structured.

Reply:
Whoops sorry, I made a mistake here. I didn't notice that I was reviewing `FindGINKO.cmake`. Apologies :-/